RockWeb/Themes/NewSpring/assets/Lava/DATA/get-permalink.lava

```diff
@@ -18,7 +18,7 @@
 {% if pccid and pccid != empty %}
     {%- comment -%}Item has a primary parent channel{%- endcomment -%}
     {% sql %}
-        SELECT 
+        SELECT
```

Comment:
You didn't really make any changes to this file ... these are all spacing. Not a big deal but you could remove this from here. We don't need this file to change if nothing really did.